workers/dockerfiles/Dockerfile.illumina

```diff
@@ -7,11 +7,15 @@ RUN apt-get update -qq
 RUN apt-get -y install apt-fast
 
 COPY workers/CRAN.gpg .
+RUN apt-key adv --keyserver keyserver.ubuntu.com --recv-keys E084DAB9
```

Comment:
Wait why is this being done this way. Aren't we supposed to do this key file like `apt-key add CRAN.gpg`?